woocommerce/woocommerce-ios
# Product creation AI: Use product name in package flow

Closes: #10739 

## Description

Use the product name from the "Add product name" screen in the package flow to generate the product title. 

Changes
- Inject product name from product creation AI flow into package flow.
- Use injected product name in package flow prompt. 

## Testing instructions

- Create a JN site with Jetpack AI plugin. (Checkbox available in JN home page to install this plugin)
- Switch to Products tab, select the "+" button to add a new product. 
- Tap on "Create a product with AI" to launch  "Add your product name" screen
- Enter a product name
- Tap on the "Use package photo" button and launch the package flow
- Upload a package photo
- Observe that AI uses the entered product name to generate the product name. 

## Screenshots

https://github.com/woocommerce/woocommerce-ios/assets/524475/5ed9051e-004b-4487-b356-3fc5b833c56c


---
